services/ui-src/src/verbiage/pages/accordion.ts

```diff
@@ -4,115 +4,118 @@ export default {
       buttonLabel: "Instructions",
       intro: [
         {
-          type: "text",
-          as: "span",
-          content: "<b>State User Instructions</b><br/><br/>As described at ",
-        },
-        {
-          type: "externalLink",
-          content: "42 CFR § 438.74",
-          props: {
-            href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-B/section-438.74",
-            target: "_blank",
-            "aria-label": "42 CFR § 438.74 (link opens in new tab)",
-          },
-        },
-        {
-          type: "text",
-          as: "span",
-          content:
-            ", states are required to report summary Medical Loss Ratio (MLR) reports to the Centers for Medicare & Medicaid Services (CMS). The summary MLR report submission coincides with the state’s submission of the annual base rate certification. The summary reports are based on the plans’ annual MLR reports to the state required under ",
-        },
-        {
-          type: "externalLink",
-          content: "42 CFR § 438.8(k)",
-          props: {
-            href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-A/section-438.8#p-438.8(k)",
-            target: "_blank",
-            "aria-label": "42 CFR § 438.8(k) (link opens in new tab)",
-          },
-        },
-        {
-          type: "text",
-          as: "span",
-          content:
-            ". If the state needs to revise MLR reports previously submitted using the online form, ask your CMS contact to reopen the related submission, which will change it to an “In revision” status. If the state needs to revise MLR reports previously submitted using the Excel Workbook or other format, reach out to your CMS contact to discuss next steps. ",
-        },
-        {
-          type: "externalLink",
-          content: "Learn more about the Medical Loss Ratio report",
-          props: {
-            href: "https://www.medicaid.gov/medicaid/managed-care/guidance/medicaid-and-chip-managed-care-reporting/index.html#MLR",
-            target: "_blank",
-            "aria-label":
-              "Learn more about the Medical Loss Ratio report (link opens in new tab)",
-          },
-        },
-        {
-          type: "text",
-          as: "span",
-          content: ". <br/> <br/> <b>CMS Admin Instructions</b>",
+          type: "p",
+          children: [
+            {
+              type: "html",
+              content:
+                "<b>State User Instructions</b><br/><br/>As described at ",
```

Comment:
are we able to apply these breaks with styling?

Reply:
💯 check it out!!